src/MainModel.elm

```diff
@@ -68,7 +72,9 @@ initialModel =
     , datePickerClosed = True
     , pickedDates = initialDates
     , pickedTracks = []
+    , pickedColumns = []
     , datesWithSessions = [ { date = DateWithoutTime 2017 1 1, sessions = initialSessions } ]
+    , host = "localhost:8000"
```

Comment:
Is there a point in setting here this as it is immediately overwritten by  init?